entry_types/scrolled/package/spec/frontend/playerControls/PlayPauseButton-spec.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import '@testing-library/jest-dom/extend-expect'
+import {render} from '@testing-library/react'
+
+import {PlayPauseButton} from 'frontend/playerControls/PlayPauseButton';
+
+describe('PlayPauseButton', () => {
+  it('shows pause icon when playing', () => {
+    const {getByTestId} = render(<PlayPauseButton isPlaying={true}/>);
+    expect(getByTestId('pause-icon')).toBeVisible();
```

Comment:
As mentioned [here](https://testing-library.com/docs/dom-testing-library/api-queries#bytestid):

> [...] it is recommended to use [test ids] only after the other queries don't work for your use case. Using data-testid attributes do not resemble how your software is used and should be avoided if possible.

The need for a testid here is a hint that the control is missing some sort of title or label. This is one of the aspects I like the most about `testing-library` compared to enzyme: it nudges you to do the more accessible thing.